seq.go

```diff
@@ -10,11 +10,11 @@ type SequenceProvider interface {
 	SequenceForAccount(aid string) (int64, error)
 }
 
-type legacyClient struct {
+type LegacyClient struct {
```

Comment:
i exported this to fix a lint complaint:
`account.go:97:15: exported func Client returns unexported type *stellarnet.legacyClient, which can be annoying to use`